FetchMigration/python/orchestrator.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import logging
+import os
+import subprocess
+
+import migration_monitor
+import pre_migration
+from migration_monitor_params import MigrationMonitorParams
+from pre_migration_params import PreMigrationParams
+
+
+__DP_EXECUTABLE_SUFFIX = "/bin/data-prepper"
+__PIPELINE_OUTPUT_FILE_SUFFIX = "/pipelines/pipeline.yaml"
+
+
+def run(dp_base_path: str, dp_config_file: str, dp_endpoint: str):
+    dp_exec_path = dp_base_path + __DP_EXECUTABLE_SUFFIX
+    output_file = dp_base_path + __PIPELINE_OUTPUT_FILE_SUFFIX
+    pre_migration_params = PreMigrationParams(dp_config_file, output_file, report=True)
+    logging.info("Running pre-migration steps...\n")
+    pre_migration_result = pre_migration.run(pre_migration_params)
+    if len(pre_migration_result.created_indices) > 0:
+        # Kick off a subprocess for Data Prepper
+        logging.info("Running Data Prepper...\n")
```

Comment:
Do you want to print out the args that you're sending to it too?

Reply:
The Data Prepper process consumes its pipeline configuration from a file, which could contain credential information. I don't think we should output the contents of this file to the logs.